main/tests/server/src/com/google/refine/grel/GrelTests.java

```diff
@@ -80,30 +81,20 @@ public void testInvalidSyntax() {
 //                "2^3" // TODO: Should this generate an error?
         };
         for (String test : tests) {
-            try {
-                MetaParser.parse("grel:" + test);
-            } catch (ParsingException e) {
-                // Test succeeded
-                continue;
-            }
-            fail("Expression failed to generate parse syntax error: " + test);
+            assertThrows(ParsingException.class, () -> MetaParser.parse("grel:" + test));
```

Comment:
```suggestion
            assertThrows("Expression failed to generate parse syntax error: " + test, ParsingException.class, () -> MetaParser.parse("grel:" + test));
```
For loop based tests, it's valuable to know which iteration failed